packages/wallet/src/services/types.ts

```diff
@@ -71,8 +73,23 @@ export interface Delegatee {
   nextNextEpoch: Cardano.StakePool | null;
 }
 
+export enum DelegationKeyStatus {
```

Comment:
```suggestion
export enum StakeKeyStatus {
```
This should be a bit more generic to handle stake pool registration, which is a separate action to delegating stake. Could also be `StakeKeyRegistrationStatus` if the specificity is desired.